apitest/src/main/java/bisq/apitest/method/GetBalanceTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.apitest.method;
+
+import bisq.proto.grpc.GetBalanceRequest;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+
+
+import bisq.apitest.OrderedRunner;
+import bisq.apitest.annotation.Order;
+
+@Slf4j
+@RunWith(OrderedRunner.class)
+public class GetBalanceTest extends MethodTest {
+
+    @BeforeClass
+    public static void setUp() {
+        try {
+            setUpScaffold("bitcoind,seednode,alicedaemon");
+
+            // Have to generate 1 regtest block for alice's wallet to show 10 BTC balance.
+            bitcoinCli.generateBlocks(1);
+
+            // Give the alicedaemon time to parse the new block.
+            MILLISECONDS.sleep(1500);
+        } catch (InterruptedException ex) {
+            fail(ex.getMessage());
+        }
+    }
+
+    @Test
+    @Order(1)
+    public void testGetBalance() {
+        var balance = grpcStubs.walletsService.getBalance(GetBalanceRequest.newBuilder().build()).getBalance();
+        assertEquals(1000000000, balance);
+    }
```

Comment:
It is an important part of the `dao-setup` for regtest/dev mode testing.     Bob & Alice are given a initial BTC & BSQ balances and dummy payment accounts so they can trade without coders / testers having to go through the motions of funding wallets from `bitcoin-cli -regest`, setting up payment account, etc.  Having to do that would make release testing even more time consuming that it already is.  

There are more details [here](https://github.com/bisq-network/bisq/blob/master/docs/dao-setup.md).

Eventually, we want to be able to test creating the DAO setup environment, but I don't think that's in scope for quite awhile.

Reply:
Do you think it would make automated tests more time consuming as well. Feels dirty not starting with a clean slate. As an alternative, we could be diligent about commenting in the test suites what the initial state is.

Reply:
I added a javadoc comment explaining why Bob & Alice have non-zero initial BTC & BSQ balances and a dummy payment account before tests start.  (See commit b4d3ea7.)